sdk/storage/azure_storage_blob/src/clients/blob_client.rs

```diff
@@ -1,2 +1,70 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+
+use crate::blob_blob_client::BlobBlobClientGetPropertiesOptions;
+use crate::blob_client::BlobClientOptions;
+use crate::models::blob_properties::{build_from_response_headers, BlobProperties};
+use crate::policies::storage_headers_policy::StorageHeadersPolicy;
+use crate::BlobClient as GeneratedBlobClient;
+use azure_core::credentials::TokenCredential;
+use azure_core::{BearerTokenCredentialPolicy, Policy, Result};
+use std::sync::Arc;
+
+pub struct BlobClient {
+    pub(crate) endpoint: String,
+    pub(crate) container_name: String,
+    pub(crate) blob_name: String,
+    pub(crate) credential: Arc<dyn TokenCredential>,
+    pub(crate) client: GeneratedBlobClient,
```

Comment:
Currently this will fail (since all warnings are errors) since we never use `credential` or `endpoint`: passed directly to underlying generated client on L47. 

I think because this information is contained in the `GeneratedBlobClient` and we maintain that, so we can likely omit storing this information directly on this struct.